Dockerfile-production

```diff
@@ -47,7 +47,7 @@ RUN sudo apt-get install -yq nodejs \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install gem sass for grunt-contrib-sass
+# Install gem sass
 RUN gem install sass
```

Comment:
When I first started using this project I was on Windows 7 & had Node v0.10.x installed; I was required to install ruby/sass with that setup. Now I'm using Windows 10 & Node v6.6.0.

What @sujeethk is saying makes sense, if it's indeed a Node version issue.

@sujeethk Just out of curiosity, what Node version are you running?